scout/adapter/mongo/hgnc.py

```diff
@@ -93,7 +96,7 @@ def hgnc_id(self, hgnc_symbol, build="37"):
             hgnc_id(int)
         """
         # LOG.debug("Fetching gene %s", hgnc_symbol)
-        query = {"hgnc_symbol": hgnc_symbol, "build": build}
+        query = {"hgnc_symbol": hgnc_symbol, "build": str(build)}
```

Comment:
Strong typing languages do have a point sometimes.. 😅

Reply:
@moonso @northwestwitch This looks safe enough; but in general, is there no more elegant way to handle this, like setting an attribute on build or whatnot?!

Reply:
I'm sure there is a better way, I was thinking using a session object which will change for every case that is accessed, but that would imply even more refactoring and I was a bit scared of changing too much, especially after the last releases.. This was a fix meant to give @bjhall something to work with now, but of course we should improve the code in the future!